signer/src/lib.rs

```diff
@@ -0,0 +1,173 @@
+#![warn(missing_docs)]
+
+//! Mina signer library for verification and signing
```

Comment:
btw, if you don't want to repeat yourself between the README and the doc here, you can also include some of the README content with this: https://github.com/o1-labs/proof-systems/blob/master/ocaml/ocaml-gen/src/lib.rs#L3

Reply:
rust is awesome